pegasus/sites.v3/code.org/views/school_info.haml

```diff
@@ -0,0 +1,102 @@
+        %option{value: SchoolInfo::SCHOOL_TYPE_PUBLIC} Public
+        %option{value: SchoolInfo::SCHOOL_TYPE_OTHER} Other
+
+    .form-group{style: "display: none;"}
+      .question State
+      %select#school-state.form-control{name: "#{form_name}[school_state]", type: "select", required: false}
+        %option{value: "", selected: true, disabled: true}
+        - get_all_states_with_dc.each do |state_abbr, state_name|
+          %option{value: state_abbr}= state_name
+
+    .form-group{style: "display: none;"}
+      .question District
+      #school-district
+
+        -# We'll apply the Selectize UI control to this input.
+        %input#school-district-id.form-control{name: "#{form_name}[school_district_id]", type: "text", required: false}
+
+        %label
+          %input#school-district-other{name: "#{form_name}[school_district_other]", type: "checkbox", value: "true", style: "margin-top: 0px"}
+          %span{style: "font-size: 12px"}
+            Other district not listed above
+
+        .form-group{style: "display: none;"}
+          .question District Name
+          %div.controls
+            %input#school-district-name.form-control{name: "school_info[school_district_name]", type: "text", required: false, style: "width: calc(100% - 15px);"}
+
+    .form-group{style: "display: none;"}
+      .question School
+      #school
+
+        -# We'll apply the Selectize UI control to this input.
+        %input#school-id.form-control{name: "#{form_name}[school_id]", type: "text", required: false}
+
+        %label
+          %input#school-other{name: "#{form_name}[school_other]", type: "checkbox", value: "true", style: "margin-top: 0px"}
+          %span{style: "font-size: 12px"}
+            Other school not listed above
+
+    #no-schools-warning{style: "display: none; margin-bottom: 5px;"}
+      .question No schools of that type were found in this district.
+
+    -# '#school-name' is outside of '#school' so that we can still show it when the school dropdown is hidden.
+    .form-group{style: "display: none;"}
+      .question
+        School Name
+      %div.controls
+        %input#school-name.form-control{name: "school_info[school_name]", type: "text", required: false, style: "width: calc(100% - 15px);"}
+
+    .form-group{style: "display: none;"}
+      .question School’s ZIP code
+      %input#school-zipcode.form-control{name: "#{form_name}[school_zip]", type: "text", required: false}
+
+    .form-group{style: "display: none;"}
+      .question
+        School Address
+      %div.controls
+        %textarea#school-address.form-control{name: "school_info[full_address]", required: false, rows: 4, style: "width: calc(100% - 15px);"}
+
+:javascript
+
+  $(document).ready(function() {
+    var options = #{existing_school_info.to_json};
+    var suppressScrolling = true;
+    options['suppressScrolling'] = suppressScrolling;
+
+    var assumeUsa = false;
+
+    if (assumeUsa) {
```

Comment:
Given that `assumeUsa` is `false`, it looks like this `if` statement will never do anything?

Reply:
Yeah. That was short-circuiting some copied code (from here: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/shared/_school_info.html.haml#L95). I wasn't sure if I was going to need to later inject assumeUsa in so I left it in place but wired to false for the moment.